dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/configuration/JobParametersConfiguration.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.configuration;
+
+
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.boot.autoconfigure.mongo.MongoProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import uk.ac.ebi.eva.dbsnpimporter.parameters.DbsnpDatasource;
+import uk.ac.ebi.eva.dbsnpimporter.parameters.Parameters;
+
+import java.util.HashMap;
+
+@Configuration
+public class JobParametersConfiguration {
+
+    private HashMap<String, JobParameter> parametersMap;
+
+    @Bean
+    JobParameters jobParameters(Parameters parameters, DbsnpDatasource dbsnpDatasource,
+                                MongoProperties mongoProperties) {
+
+        parametersMap = new HashMap<>();
+
+        addParameter("assembly", parameters.getAssembly());
```

Comment:
I'm fine with keeping the strings as they are, if as you say they are not going to be used anywhere else. If we discover that is not the case in the future we can extract them to constants, but I'm not sure how beneficial it would be right now.

Reply:
exactly what I thought when I did this.